### PR TITLE
Update network-plugins.md

### PR DESCRIPTION
**Pull request by jiaj12**
update zh-trans to match master branch, the orig seems to be based on release-1.14

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -134,11 +134,13 @@ CNI 网络插件支持 `hostPort`。 您可以使用官方 [portmap](https://git
 <!--
 #### Support traffic shaping
 
+**Experimental Feature**
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:137]: this line is not translated?

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 2]
```diff
@@ -147,7 +149,7 @@ CNI 网络插件还支持 pod 入口和出口流量整形。
 也可以使用您自己的具有带宽控制功能的插件。
 
 如果您想要启用流量整形支持，你必须将 `bandwidth` 插件添加到 CNI 配置文件
-（默认是 `/etc/cni/net.d`）。
+（默认是 `/etc/cni/net.d`）并保证该二进制包含在您的 CNI 的 bin 文件夹内 (默认为 `/opt/cni/bin`)。
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:152]: binary 在这个上下文中是可执行文件的意思

Comment (tengqm) [reply content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:154]: how about this?

**Comment by tengqm** [comment]
@jiaj12 Can you squash the commits?

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:137]: maybe line 147?

**Comment by chenrui333** [comment]
ping @jiaj12

**Comment by jiaj12** [comment]
still got problem on squash. After I squash recent 3 commit to 1 and commit to local patch-31, I got rejected when pushing to origin/patch-31. How to  overwrite the remote history? push to another branch?

**Comment by tengqm** [comment]
After you have squashed your local commits, do a force push to  your fork: `git push -f`.

**Comment by jiaj12** [comment]
> After you have squashed your local commits, do a force push to your fork: `git push -f`.

How about this time?

**Comment by tengqm** [comment]
Worked!
/lgtm
/approve

**Pull request merged**